### PR TITLE
Adding .deb installer

### PR DESCRIPTION
**Pull request by borisfom**
Most changes are in debian/ directory. Fixed tests not to depend on nvidia-ml.

### CODE REVIEW

**Comment by borisfom** [comment]
Please hold on integrating this: I am rebuilding the installer and will update the PR shortly.

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Maintainer: Boris Fomitchev <bfomitchev@nvidia.com>
+Priority: optional
+Build-depends: debhelper(>=9), nvidia-352-dev, cuda-toolkit-7-0,
+               cuda-curand-dev-7-0, cuda-cublas-dev-7-0, cuda-cudart-dev-7-0, cuda-ld-conf-7-0
+Standards-Version: 3.9.5
+
+Package: nccl
+Section: universe/tools
+Architecture: amd64
+Depends: ${misc:Depends}, ${shlibs:Depends},  nvidia-352,
```
Comment (lukeyeager) [inline debian/control]: Is it necessary to depend on a specific driver version? What if they installed with a RUN file?

Comment (Kaixhin) [inline debian/control]: Can confirm that I use the RUN file and know plenty of others do. A range of drivers should ideally be supported.

Comment (lukeyeager) [inline debian/control]: I'd vote for moving `nvidia-352` to the `Suggests` field - i.e. not installed by default.

**Comment by borisfom** [comment]
Superseded by Luke's request

**Pull request closed without merging**